tests/sharingTest.js

```diff
@@ -1,26 +1,32 @@
+import { MatchersV3, XmlBuilder } from '@pact-foundation/pact/v3'
+
 describe('Main: Currently testing file/folder sharing,', function () {
   const config = require('./config/config.json')
+  config.testFiles = [
+    'test space and + and #.txt',
+    'test.txt',
+    'hello world.txt'
+  ]
+
+  config.testFilesPath = [
+    '%2Ftest+space+and+%2B+and+%23.txt',
+    '%2Ftest.txt',
+    '%2Fhello+world.txt'
+  ]
```

Comment:
I added this change because with pact when we add an interaction with a query for `'文件.txt'` it doesn't work.

@individual-it I think it is a bug with pact. do you have any idea.

Reply:
can you try to reproduce within the pact-js examples (v3 branch) https://github.com/pact-foundation/pact-js/tree/feat/v3.0.0/examples

Reply:
I've added comments with links to the issue